src/scripting/lua_map.cc

```diff
@@ -2110,6 +2110,12 @@ int LuaMapObjectDescription::helptexts(lua_State* L) {
 	return 1;
 }
 
+// Sets the description pointer for a MapObjectDescription type
+#define UNPERSIST_DESCRIPTION(L, type, name)                                                       \
```

Comment:
No way to use an inline method #define Macros are so oldschool (and error prone)